doc/api/errors.md

```diff
@@ -818,9 +818,11 @@ HTTP/1 connection specific headers are forbidden to be used in HTTP/2
 requests and responses.
 
 <a id="ERR_HTTP2_INVALID_HEADER_VALUE"></a>
-### ERR_HTTP2_INVALID_HEADER_VALUE
+### ERR_HTTP_INVALID_HEADER_VALUE
```

Comment:
This line is duplicated

Reply:
Actually it's accidentally changed..we need to keep the documentation for `ERR_HTTP2_INVALID_HEADER_VALUE` because it's out there already.

Reply:
If the codebase doesn't raise an error with this code though, should it be removed? 😬